samples/wifi/sta/README.rst

```diff
@@ -0,0 +1,83 @@
+.. _wifi_sta_sample:
+
+Wi-Fi: Sta
+###########
+
+.. contents::
+   :local:
+   :depth: 2
+
+This sample allows Nordic Semiconductor's Wi-Fi chipset to connect with Access point.
+
+Requirements
+************
+
+The sample supports the following development kit:
+
+.. table-from-sample-yaml::
+
+Overview
+********
+
+This sample can perform Wi-Fi operations "Connect and Disconnect" in the 2.4GHz and 5GHz bands depending on the capabilities supported in the underlying chipset.
+
+Using this sample, the development kit can associate with any Wi-Fi capable access point in :abbr:`STA (Station)` mode.
+
+Building and running
+********************
+
+.. |sample path| replace:: :file:`samples/wifi/sta`
+
+.. include:: /includes/build_and_run_ns.txt
+
+Currently, the following configurations are supported:
+
+* 7002 DK + QSPI
+
```

Comment:
Add details of how to configure the sample to connect to the desired Access Point - basically where to add the SSID, Password etc.